### PR TITLE
Fix Travis example tests for release-1.13

### PR DESCRIPTION
**Pull request by tengqm**
Closes: #13121
xref: #13122

### CODE REVIEW

**Comment by DanyC97** [comment]
Fixes https://github.com/kubernetes/website/issues/13121

File: content/en/examples/pods/pod-projected-svc-token.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ spec:
     volumeMounts:
     - mountPath: /var/run/secrets/tokens
       name: vault-token
+  serviceAccountName: acct
```
Comment (DanyC97) [inline content/en/examples/pods/pod-projected-svc-token.yaml]: good catch @tengqm , thank you !

**Comment by DanyC97** [comment]
great work @tengqm 💯 
/lgtm

**Comment by cody-clark** [comment]
Thanks, @tengqm, for getting this out so quickly!

/approve

**Pull request merged**